src/QtDBMigration.cpp

```diff
@@ -43,10 +45,24 @@ class QtDBMigrationPrivate
     AbstractConfig *m_config;
 };
 
+void QtDBMigrationPrivate::setConfig(QString path) {
+    if (QFileInfo::exists(path))
```

Comment:
Are you sure this is correct?

Have you tested the current functionality still works?

Reply:
I see, it checks if *file* exists.

Ok.

Reply:
Irelevant.